### PR TITLE
Add offer related properties to message interaction mixin

### PR DESCRIPTION
**Pull request by usankara**
Adding following offer related properties to message interaction mixin.

- offerId
- placementId
- propositionId
- propositionTime
- placementCount

### CODE REVIEW

**Comment by usankara** [comment]
@guayang-adobe , @sawhneyh, @prabhum2 please review.

File: extensions/adobe/experience/customerJourneyManagement/message-interaction.schema.json [hunk 1]
```diff
@@ -51,6 +51,31 @@
...
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/messageInteraction/propositionTime": {
+          "title": "Proposition Time",
+          "type": "string",
+          "description": "Time at which the proposition was generated"
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/messageInteraction/placementCount": {
+          "title": "Placement Count",
+          "type": "integer",
+          "description": "Count of placements"
```
Comment (guayang-adobe) [inline extensions/adobe/experience/customerJourneyManagement/message-interaction.schema.json:78]: "Count of placements included in the offer proposition"?

**Review by guayang-adobe — COMMENTED** [review]
Let's have "offer" in the descriptions, e.g.  "Unique Identifier for Offer Placement"

**Review by guayang-adobe — APPROVED** [review]

**Comment by mjasrotia** [comment]
@usankara , we generally name the identity properties with "ID" and not "Id". I have changed the property names here.

**Comment by usankara** [comment]
thanks @mjasrotia for correcting the property names.

**Review by prabhum2 — APPROVED** [review]

**Comment by mjasrotia** [comment]
@usankara , Is this PR still under review from your team? I see you are still making changes on this

**Comment by usankara** [comment]
@mjasrotia, yes, there was a change of requirement at the last minute. we are mostly done with the changes unless @sawhneyh has some review comments. I think we need to wait till Monday EOD PST for Himanshu's comments.

File: extensions/adobe/experience/customerJourneyManagement/message-interaction.schema.json [hunk 2]
```diff
@@ -50,7 +50,32 @@
         "https://ns.adobe.com/experience/customerJourneyManagement/messageInteraction/label": {
           "title": "Tracker URL Label",
           "type": "string",
-          "description": "Human-Friendly label"
+          "description": "Human-Friendly label."
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/messageInteraction/offerID": {
```
Comment (sawhneyh) [inline extensions/adobe/experience/customerJourneyManagement/message-interaction.schema.json:55]: Should we create a subsection for Offer related fields in messageInteraction object.
Also, we should update messagefeedback schema also for Offer related fields.

Comment (usankara) [reply extensions/adobe/experience/customerJourneyManagement/message-interaction.schema.json]: ok, did the changes to separate out offers section.
messagefeedback schema changes need to be taken up separately by the right owner (efs team?) as a separate PR.

**Comment by mjasrotia** [comment]
@usankara , please let us know when this is good to go.
@prabhum2 , please do not merge this yet..

File: extensions/adobe/experience/customerJourneyManagement/message-interaction.schema.json (hunk 2 above)
Comment (usankara) [reply extensions/adobe/experience/customerJourneyManagement/message-interaction.schema.json]: @mjasrotia, yes, this is good to go. Thank you!

**Comment by usankara** [comment]
@mjasrotia, yes, this is good to go. Thank you!

**Comment by usankara** [comment]
corrected path for offer fields, as pointed out by @guayang-adobe

**Review by sawhneyh — APPROVED** [review]

**Comment by usankara** [comment]
@mjasrotia, @prabhum2  , we are truly good to go now :-)

**Review by mjasrotia — APPROVED** [review]

**Pull request merged**